docker/Dockerfile

```diff
@@ -81,15 +76,22 @@ ENV ZEBRA_SKIP_IPV6_TESTS=${ZEBRA_SKIP_IPV6_TESTS:-1}
 # This environment setup is almost identical to the `runtime` target so that the
 # `tests` target differs minimally. In fact, a subset of this setup is used for
 # the `runtime` target.
-
 ARG UID
+ENV UID=${UID}
 ARG GID
+ENV GID=${GID}
```

Comment:
Why do we need these?

Reply:
Just for convenience, in case the user builds with custom UID/GID and then needs those variables available in the container.